letsencrypt/client/init.sls

```diff
@@ -102,9 +102,66 @@ certbot_cron:
   file.managed:
     - name: /etc/cron.d/certbot
     - source: salt://letsencrypt/files/cron
+    - template: jinja
     - require:
       - cmd: certbot_installed
 
 {%- endif %}
 
+{%- if client.hook %}
+certbot_deploy_hook:
+    file.managed:
+        - name: /usr/local/lib/certbot/hooks/deploy-hook
```

Comment:
Isn't better to use /etc/certbot as a location for these hooks instead of introducing this path?

Reply:
You're probably right, I forgot to look into that. I don't have an `/etc/certbot`, is that created automatically by something? Is `/etc/certbot/hooks` the correct place for them?

Reply:
Sorry, it's `/etc/letsencrypt/`
Didn't find any recommendation but that location probably makes more sense.

Reply:
There _is_ a `renewal-hooks` directory but I worry that some other script automatically runs hooks out of there under some circumstances. Do you know if that's the case? Is it safe to drop the deploy hook into `/etc/letsencrypt/renewal-hooks/deploy`?

Reply:
Unfortunately I don't know :-/ Would prefer using some custom directory, eg. `scripts`